src/sanghwa/memberSet.java

```diff
@@ -0,0 +1,96 @@
+package sanghwa;
+
+import jun.Member;
+
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class memberSet {
+    public static HashSet<Member> set = new HashSet<>();
+
+    public boolean addMember(String name, int age, int id) {
+        Member member = new Member(name, age, id);
+        boolean isIn = set.add(member);
+        if (isIn){
+            return true;
+        }
+        return false;
+    }
+    public static int getSize(){
+        return set.size();
+    }
+
+    public  boolean removeMember(int id){
```

Comment:
```suggestion
    public boolean removeMember(int id) {
```

public과 boolean(반환 타입)에 사이에 불필요한 공백이 있어요!
또한 매개변수 다음오 오는 중괄호는 공백을 통해서 띄워주는게 관례에요
리팩토링 해보세요!